src/components/switch.module.scss

```diff
@@ -0,0 +1,70 @@
+$switch-width: 80px;
+$switch-height: var(--button-height);
+$inner-width: 50px;
+$unchecked-colour: #813030;
+$unchecked-colour-accent: #a32121;
+$checked-colour: #308130;
+$checked-colour-accent: #21a321;
+
+.switch {
+    display: inline-block;
+    position: relative;
+    width: $switch-width;
+    height: $switch-height;
+    background-color: var(--mid-gray);
+    border: 2px solid $unchecked-colour-accent;
+    border-radius: calc(#{$switch-height} / 2);
```

Comment:
I think it would be good to match this up a bit more to the general button/input style.

i.e. non rounded.

Maybe get a bit of input from Sam L.

Happy to leave it as it is but anticipate tweaks 👍